.github/actions/pnpm/action.yml

```diff
@@ -0,0 +1,16 @@
+name: Pnpm
```

Comment:
This is not needed, can inline this.

Reply:
Or maybe refactor the other scripts later?

Reply:
Yes, refactor later